src/reactapp/src/utils/customer.js

```diff
@@ -0,0 +1,24 @@
+import _get from 'lodash.get';
+import { _keys } from '.';
+
+export function capitalize(str) {
+  if (!str) {
```

Comment:
could be slightly simplified:
```
if (typeof str !== 'string') {
  return ''
}
return str.charAt(0).toUpperCase() + str.slice(1)
```

Reply:
I will be including this change in the next pull request. thanks.